draft-ietf-homenet-front-end-naming-delegation.txt

```diff
@@ -1158,6 +1171,13 @@ Internet-Draft                public-names                     July 2021
    fact, the Public Authoritative Server(s) may use Automating DNSSEC
```

Comment:
line 1166: - manually where?

Reply:
We specified the DS RRSet can be updated manually in the parent zone.